test/test_resource.rs

```diff
@@ -0,0 +1,35 @@
+use nix::sys::resource::{Resource, getrlimit, setrlimit};
+
+#[test]
+pub fn test_resource_limits_nofile() {
```

Comment:
Can you summarize what this test is doing? Makes it easier to fix or modify them later.